src/api.ts

```diff
@@ -704,47 +704,175 @@ export default class API extends EventEmitter {
     const fillPrice = (quoteQuantity / baseQuantity).toFixed(
       marketInfo.pricePrecisionDecimals
     )
+    const redis_members: any = {
+      score: fillPrice,
+      value: JSON.stringify({
+        "selectresult": selectresult,
+        "fillOrder": fillOrder,
+        "ws": ws
```

Comment:
I don't think storing the WS in Redis is a good idea. There's a potential for the JSON to be circular and fail to store. Storing the WS UUID and referencing it later would be better.

Reply:
changed with https://github.com/ZigZagExchange/backend/pull/54/commits/db991f4bbaa299071bcbab06e539cb02771625b0